man/man1/oci-kvm.1

```diff
@@ -59,6 +77,19 @@ The
 .B oci-kvm create-pool
 subcommand creates a storage pool to host guest virtual disks. Pools can be created upon local disk, block storage of FSS space.
```

Comment:
Pools can be created on block devices or FSS mounted file systems

Reply:
I've committed the word 'mounted' as Libvirt actually mount it . the filesystem do not have to 
be mounted